test/parallel/test-runner-exit-code.js

```diff
@@ -54,5 +54,5 @@ if (process.argv[2] === 'child') {
   assert.strictEqual(child.signal, null)
 
   runAndKill(fixtures.path('test-runner', 'never_ending_sync.js')).then(common.mustCall())
-  runAndKill(fixtures.path('test-runner', 'never_ending_async.js')).then(common.mustCall())
+  runAndKill(fixtures.path('test-runner', 'never_ending_async.js')).then(typeof AbortSignal === 'undefined' ? undefined : common.mustCall())
```

Comment:
Turns out that's the only change on files that are from nodejs/node that needs to be made to make the tests pass, maybe we don't even need to release a semver-major after all?

Reply:
Nevermind, we can workaround that as well – that's very hacky, but as long as the tests pass 🤷‍♂️